projects/frontend/cicd/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+
+RUN apt-get update && apt-get install -y \
+    curl \
```

Comment:
I prefer we avoid pinning versions despite what Codacy says and use latest versions to reduce future maintenance.  

That's not against best practices if we use "cache busting" which we do (apt-get install update does that) (reading https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get)

Reply:
Where do see the codacy warning?

Reply:
I agree with your above comment. I will not be addressing this codacy issue before merging.